app/data/census_individual.json

```diff
@@ -125,11 +44,11 @@
                 },
                 {
                     "id": "sex",
-                    "title": "John Doe",
+                    "title": "{{ answers.person_full_name }}",
```

Comment:
Is this correct shouldn't it be something like {{answers.person_full_name[current.group_instance]}}?